src/matter/cluster/AdminCommissioningCluster.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { TlvByteString, TlvField, TlvObject, TlvUInt16, TlvUInt32 } from "@project-chip/matter.js";
+import { Cluster, Command, TlvNoArguments, TlvNoResponse, OptionalCommand } from "./Cluster";
+
+export const AdminCommissioningCluster = Cluster({
```

Comment:
Add splec ref Core specs 11.18

Reply:
This cluster is used to trigger a Node to allow a new Administrator to commission it. It defines Attributes, Commands and Responses needed for this purpose.

Reply:
Added